libs/coin-modules/coin-multiversx/src/bridge.integration.test.ts

```diff
@@ -135,11 +132,15 @@ const elrond: CurrenciesData<Transaction> = {
   ],
 };
 
-const dataset: DatasetTest<Transaction> = {
+export const dataset: DatasetTest<Transaction> = {
   implementations: ["js"],
   currencies: {
     elrond,
   },
 };
 
-testBridge(dataset);
+describe("Elrond bridge", () => {
```

Comment:
Change test suite name to MultiversX bridge

Reply:
A full rebranding will be done in a later PR.